README.md

```diff
@@ -1,6 +1,6 @@
 Authors Haven - A Social platform for the creative at heart.
 
-[![Build Status](https://travis-ci.com/andela/vidar-ah-backend.svg?branch=develop)](https://travis-ci.com/andela/vidar-ah-backend) [![Coverage Status](https://coveralls.io/repos/github/andela/vidar-ah-backend/badge.svg?branch=develop)](https://coveralls.io/github/andela/vidar-ah-backend?branch=develop) [![](https://img.shields.io/badge/Protected_by-Hound-a873d1.svg)](https://houndci.com)
+[![Build Status](https://travis-ci.com/andela/vidar-ah-backend.svg?branch=develop)](https://travis-ci.com/andela/vidar-ah-backend) [![Coverage Status](https://coveralls.io/repos/github/andela/vidar-ah-backend/badge.svg?branch=develop)](https://coveralls.io/github/andela/vidar-ah-backend?branch=ft-create-user-article-%23164139686) [![](https://img.shields.io/badge/Protected_by-Hound-a873d1.svg)](https://houndci.com)
```

Comment:
@olajide1234 why are you changing the branch here?

Reply:
The code coverage results were configured for the `ft-create-user-article-23164139686` branch alone before. @kleva-j advised updating this to make sure the coverage result is for the develop branch.

Reply:
@olajide1234 I don't quite understand what you mean here. code coverage should work for all branches when configured. The branch referenced in the README dictates which branch's coverage is shown.

Reply:
Updated and resolve now. Thanks.